### ISSUE TITLE
Adherence to SQL code style guide

### ISSUE THREAD

**Issue by schuemie** [OHDSI/QueryLibrary#5]
Currently the SQL doesn't adhere to [our code style](http://www.ohdsi.org/web/wiki/doku.php?id=development:ohdsi_code_style_for_sql). For example, [this query](https://github.com/OHDSI/QueryLibrary/blob/master/inst/shinyApps/QueryLibrary/queries/condition/C03.md) uses an odd mixture of camelCase and snake_case (should be only snake_case).

If we want people to adhere to the code style, we should start by teaching them through tools like these ;-)

**Comment by PRijnbeek** [comment]
Yes agree all these queries need to be reviewed with the style guide in mind. Suggest we add an About page that refers to the guidelines?

**Comment by schuemie** [comment]
Yes, I think that's a good idea.
